Replace controller with control node (#352)
ansible/ansible-documentation: docs/docsite/rst/getting_started_ee/introduction.rst

Content separation
^^^^^^^^^^^^^^^^^^

> In situations when there is an Ansible control node or a tool such as Ansible AWX/control node used by several users, they might want separate their content to avoid configuration and dependency conflicts.

Reviewer: ```suggestion
In situations when there is an Ansible control node or a tool such as Ansible AWX/Controller used by several users, they might want separate their content to avoid configuration and dependency conflicts.
```

Reviewer: This is another place where it's referring to the Ansible Controller product so this suggested edit backs out the change to control node.

* `Ansible Builder <https://ansible-builder.readthedocs.io/en/stable/>`_
* `Ansible Navigator <https://ansible-navigator.readthedocs.io/>`_

> * `Ansible AWX <https://docs.ansible.com/automation-control node/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_

Reviewer: ```suggestion
* `Ansible AWX <https://ansible.readthedocs.io/projects/awx/en/latest/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
```

Reviewer: I'm just taking the opportunity to fix a URL in this one :-)  We just published new AWX docs.

> * `Automation Controller <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_

Reviewer 2: I would just remove the Automation Controller "latest" link here because that is going to go away.

> * `Automation Controller <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_

Reviewer: ```suggestion
```

> * `Automation control node <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_

Reviewer: ```suggestion
* `Automation Controller <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
```

Reviewer: This one is the actual Automation Controller product.
